google-startup-scripts/usr/lib/systemd/system/google-shutdown-scripts.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Google Compute Engine user shutdown scripts
+Before=shutdown.target
+DefaultDependencies=true
+Wants=local-fs.target network-online.target network.target
```

Comment:
I don't actually know if you need this Wants line. Worth testing with and without.